client/extensions/wp-super-cache/contents-tab.js

```diff
@@ -224,7 +223,7 @@ class ContentsTab extends Component {
 							onClick={ this.deleteCache }>
 							{ translate( 'Delete Cache' ) }
 						</Button>
-						{ isMultisite &&
+						{ site && site.is_multisite &&
```

Comment:
When only using an attr or two from `site`, I usually prefer using more fine-grained selectors in the `connect` statement (there are plenty `site` related ones -- [`isJetpackSiteMultiSite`](https://github.com/Automattic/wp-calypso/blob/ef40e66e73a2bde0e0e791ebdb50b8c6f7862205/client/state/sites/selectors.js#L742) in this case). (Among other things, this has the advantage of not causing re-renders when the `site` object reference changes.) Again, just FYI, no need to change here.